actions.yaml

```diff
@@ -11,18 +11,6 @@ get-password:
 pre-upgrade-check:
   description: Run necessary pre-upgrade checks before executing a charm upgrade.
 
-set-password:
```

Comment:
No more set-password is possible?

Reply:
No, we actually never could set a password (so far).

We can:
 - either extend Opensearch Dashboards to allow for setting Opensearch user passwords
 - or stick to the current approach were user passwords are to be set directly on Opensearch

(Note that Dashboards is only using Opensearch users. It doesn't have any users on its own.)

Reply:
Yes, makes sense to remove it then